sql/base/characters_database.sql

```diff
@@ -1,8 +1,8 @@
--- MySQL dump 10.13  Distrib 5.6.26, for Win64 (x86_64)
```

Comment:
I see many unrelated changes in the full sql dump - charsets/unsigned on some fields